client/app/services/events.js

```diff
@@ -1,5 +1,7 @@
 import recordEvent from "@/services/recordEvent";
 
+// ANGULAR_REMOVE_ME delete this file - it replaced with recordEvent
```

Comment:
I think it's time to remove this file (and replace usages with `recordEvent` of course) :slightly_smiling_face:

Reply:
I think the only place that has those events without `recordEvent` is the Angular query pages, the comment seemed worth it, since the usages will be removed as soon as the react version gets merged :slightly_smiling_face:.